Cython/Compiler/Builtin.py

```diff
@@ -193,10 +193,6 @@ def declare_in_type(self, self_type):
     #('map',       "",     "",      ""),
     #('max',       "",     "",      ""),
     #('min',       "",     "",      ""),
-    BuiltinFunction('next',       "O",    "O",     "__Pyx_PyIter_Next",
-                    utility_code = iter_next_utility_code),   # not available in Py2 => implemented here
-    BuiltinFunction('next',      "OO",    "O",     "__Pyx_PyIter_Next2",
-                    utility_code = iter_next_utility_code),  # not available in Py2 => implemented here
```

Comment:
These are inline functions for performance reasons. Please keep them.

Reply:
Reverted these changes (in 4784987)